dags/burnham.py

```diff
@@ -0,0 +1,147 @@
+    :param str gke_namespace:                   GKE cluster namespace
+    :param str docker_image:                    Docker image to use
+    :param Dict[str, Any] kwargs:               Additional kwargs for GKEPodOperator
+
+    :return: GKEPodOperator
+    """
+    kwargs["name"] = kwargs.get("name", task_id.replace("_", "-"))
+
+    env_vars = {
+        "BURNHAM_PLATFORM_URL": BURNHAM_PLATFORM_URL,
+        "BURNHAM_TEST_RUN": burnham_test_run,
+        "BURNHAM_TEST_NAME": burnham_test_name,
+    }
+
+    if burnham_spore_drive is not None:
+        env_vars["BURNHAM_SPORE_DRIVE"] = burnham_spore_drive
+
+    # TODO: Update project for Docker image
+    return GKEPodOperator(
+        task_id=task_id,
+        gcp_conn_id=gcp_conn_id,
+        project_id=GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id).project_id,
+        location=gke_location,
+        cluster_name=gke_cluster_name,
+        namespace=gke_namespace,
+        image="gcr.io/<PROJECT>/burnham:latest",
+        image_pull_policy="Always",
+        env_vars=env_vars,
+        arguments=burnham_missions,
+        owner=DAG_OWNER,
+        email=DAG_EMAIL,
+        **kwargs
+    )
+
+
+default_args = {
+    "owner": DAG_OWNER,
+    "email": DAG_EMAIL,
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "start_date": datetime.datetime(2020, 6, 15),
+    "depends_on_past": False,
+    "retries": 0,
+}
+
+with models.DAG(
+    "burnham", schedule_interval="@daily", default_args=default_args,
+) as dag:
+
+    # TODO: Update test information
+    burnham_test_run = "TEST RUN"
+    burnham_test_name = "TEST NAME"
+
+    client1 = new_burnham_operator(
+        task_id="client1",
+        burnham_test_run=burnham_test_run,
+        burnham_test_name=burnham_test_name,
+        burnham_missions=["MISSION G: FIVE WARPS, FOUR JUMPS", "MISSION C: ONE JUMP"],
+        burnham_spore_drive="tardigrade",
+    )
+
+    client2 = new_burnham_operator(
+        task_id="client2",
+        burnham_test_run=burnham_test_run,
+        burnham_test_name=burnham_test_name,
+        burnham_missions=[
+            "MISSION A: ONE WARP",
+            "MISSION B: TWO WARPS",
+            "MISSION D: TWO JUMPS",
+            "MISSION E: ONE JUMP, ONE METRIC ERROR",
+            "MISSION F: TWO WARPS, ONE JUMP",
+            "MISSION G: FIVE WARPS, FOUR JUMPS",
+        ],
+        burnham_spore_drive="tardigrade-dna",
+    )
+
+    client3 = new_burnham_operator(
+        task_id="client3",
+        burnham_test_run=burnham_test_run,
+        burnham_test_name=burnham_test_name,
+        burnham_missions=["MISSION A: ONE WARP", "MISSION B: TWO WARPS"],
+        burnham_spore_drive=None,
+    )
+
+    # TODO: Update SQL for Sensor
+    # Wait for 10 pings with this test run ID in live tables
+    wait_for_data = BigQuerySQLSensorOperator(
+        task_id="wait_for_data",
+        sql="TODO",
+        bigquery_conn_id=DEFAULT_GCP_CONN_ID,
+        use_legacy_sql=False,
+    )
```

Comment:
We'll probably want to configure a `timeout` parameter here, so that the DAG fails in the case data never arrives rather than sitting in the waiting state forever.

Reply:
Added a timeout of 4 hours. Please let me now if that makes sense.

Reply:
This will be querying the live data? We could probably push this down to an hour. Rarely do we have delay longer than that.

Reply:
Yes, from what I understand we can query live data. I'll change this to one hour!